woocommerce/woocommerce-gateway-stripe
# refactor: general settings section styles

# Changes proposed in this Pull Request:

Adjusting the settings so they slightly align more closely to all the other styles.
Before:
![Screen Shot 2021-09-13 at 10 33 19 AM](https://user-images.githubusercontent.com/273592/133114261-aed01911-6847-4c9c-8537-f34ed16c1ffa.png)


After:
![Markup 2021-09-13 at 10 56 00](https://user-images.githubusercontent.com/273592/133116831-acfb30c4-b87c-460c-a190-1c4e2dcaf17c.png)




# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings
- The UI for the "General settings" section has changed
